container/go/pkg/compat/overrideConfig.go

```diff
@@ -0,0 +1,453 @@
+		}
+		log.Printf("The first createTime is: %v", createTime)
+		// If creationTime is parsable as a floating point type, assume unix epoch timestamp.
+		// otherwise, assume RFC 3339 date/time format.
+		unixTime, err = strconv.ParseFloat(createTime, 64)
+		log.Printf("the unixTime after strconv is: %f", unixTime)
+		// Assume RFC 3339 date/time format. No err means it is parsable as floating point.
+		if err == nil {
+			log.Println("nani")
+			// Ensure that the parsed time is within the floating point range.
+			// Values > 1e11 are assumed to be unix epoch milliseconds.
+			if unixTime > 1.0e+11 {
+				unixTime = unixTime / 1000.0
+			}
+			// Construct a RFC 3339 date/time from Unix epoch.
+			sec, dec := math.Modf(unixTime)
+			log.Printf("sec: %v, dec: %v", sec, dec)
+			creationTime = time.Unix(int64(sec), int64(dec*(1e9))).UTC()
+		} else {
+			creationTime, err = time.Parse(time.RFC3339, createTime)
+			if err != nil {
+				return errors.Wrapf(err, "failed to parse the %s into float, so assuming RFC3339", createTime)
+			}
+		}
+	}
+	log.Printf("the assigned CreationTime is: %v", creationTime)
+	overrideInfo.ConfigFile.Created = v1.Time{creationTime}
+
+	if overrideInfo.NullEntryPoint == "True" {
+		overrideInfo.ConfigFile.Config.Entrypoint = nil
+	} else if len(overrideInfo.Entrypoint) > 0 {
+		for i, entry := range overrideInfo.Entrypoint {
+			stampedEntry, err := Stamp(entry, overrideInfo.StampInfoFile)
+			if err != nil {
+				return errors.Wrapf(err, "Unable to perform substitutions to Env variable %s", entry)
+			}
+			overrideInfo.Entrypoint[i] = stampedEntry
+		}
+		overrideInfo.ConfigFile.Config.Entrypoint = overrideInfo.Entrypoint
+	}
+
+	if overrideInfo.NullCmd == "True" {
+		log.Println("hellooooo")
+		overrideInfo.ConfigFile.Config.Cmd = nil
+	} else if len(overrideInfo.Command) > 0 {
+		for i, cmd := range overrideInfo.Command {
+			stampedCmd, err := Stamp(cmd, overrideInfo.StampInfoFile)
+			if err != nil {
+				return errors.Wrapf(err, "Unable to perform substitutions to Env variable %s", cmd)
+			}
+			overrideInfo.Command[i] = stampedCmd
+		}
+		overrideInfo.ConfigFile.Config.Cmd = overrideInfo.Command
+	}
+
+	log.Printf("the stamped Command: %v", overrideInfo.Command)
+
+	stampedUser, err := Stamp(overrideInfo.User, overrideInfo.StampInfoFile)
+	if err != nil {
+		errors.Wrapf(err, "Unable to perform substitutions to user %s", overrideInfo.User)
+	}
+	overrideInfo.ConfigFile.Config.User = stampedUser
+
+	environMap := keyValueToMap(overrideInfo.Env)
+	for key, valToBeStamped := range environMap {
+		stampedValue, err := Stamp(valToBeStamped, overrideInfo.StampInfoFile)
+		if err != nil {
+			return errors.Wrapf(err, "Error stamping value %s", valToBeStamped)
+		}
+		environMap[key] = stampedValue
+	}
+	// perform any substitutions of $VAR or ${VAR} with environment variables.
+	if len(environMap) != 0 {
+		var baseEnvMap map[string]string
```

Comment:
`baseEnvMap := make(map[string]string)`

Can just remove the else clause below.